### DISCUSSION TITLE
我们能否定义一种与通信方式无关的协议

### DISCUSSION THREAD

**Discussion by frg2089** [howmanybots/onebot#242] [想法 / 建议]
我们能否定义一种与通信方式无关的协议？

像 JsonRPC 就是一种与通信方式无关的协议。

有了这样的协议，我们就可以将它应用到任意一种通信方式上，而不是仅仅局限在HTTP协议上。

**Comment by stdrc** [comment]
你想说的就是 OneBot Connect 的“数据协议”，数据协议本身就是和通信方式解耦的

**Comment by stdrc** [comment]
“动作请求”就是 jsonrpc request，“动作响应”就是 response
